src/NodaTime/CalendarSystem.cs

```diff
@@ -247,7 +253,28 @@ internal static CalendarSystem ForOrdinalUncached([Trusted] CalendarOrdinal ordi
             Preconditions.CheckArgumentRange(nameof(monthNumbering), (int) monthNumbering, 1, 2);
             return HebrewCalendars.ByMonthNumbering[((int) monthNumbering) - 1];
         }
-        
+
+        /// <summary>
+        /// Returns the Wondrous (Badí') calendar, as described at https://en.wikipedia.org/wiki/Badi_calendar. 
+        /// This is a purely solar calendar with years starting at the vernal equinox.
+        /// </summary>
+        /// <remarks>
+        /// <para>The Wondrous calendar was developed and defined by the founders of the Bahá'í Faith in the mid to late
+        /// 1800's A.D. The first year in the calendar coincides with 1844 A.D. Years are labeled "B.E." for Bahá'í Era.</para>
+        /// <para>A year consists of 19 months, each with 19 days. Each day starts at sunset. Years are grouped into sets
```

Comment:
Mention that Noda Time doesn't model days starting at sunset? (There may be something appropriate for the Hebrew calendar that could be copied.)

Reply:
Sure.